proposals/security_and_privacy.md

```diff
@@ -0,0 +1,277 @@
+IoT protocols or security schemes via its built-in extension mechansisms).
+In addition, we define a general security testing procedure for WoT implementations in the
+[WoT Security Testing Plan](https://github.com/w3c/wot-security-testing-plan).
+
+## Questions and Answers
+
+### Does this specification deal with personally-identifiable information? 
+
+Indirectly.  The WoT Thing Description describes IoT devices and may include
+information such as location and type.  If these devices can be associated
+with an owner, then it may be possible to infer information about the
+owner.  For example, if they have a baby monitor, they may have a baby,
+are probably in a certain age range, and so forth.
+
+The WoT Thing Description (TD) also allows extensibility via the inclusion
+of custom vocabularies.  Although the WoT standard itself does not
+have any requirement for PII, it is possible an extension might.  We
+consider this aspect of PII inclusion to be out of scope.
+
+As for the indirect risk, we recommend that as a precaution a 
+WoT Thing Description should be treated _as if_ it contained PII
+and be stored, cached, and transmitted accordingly.  For example,
+we recommend that it only be distributed to authorized and authenticated
+users (for example, via a directory service or to a registered owner)
+and only be cached for a limited time.
+
+### Does this specification deal with high-value data? 
+
+Yes.  A WoT Thing may both require credentials to be accessed and
+use credentials to access other devices.  However, these are generally
+to secure M2M communication and are not user credentials.  We 
+define an architecture for managing sensitive credentials
+securely.
+
+The WoT architecture deals with the operational phase of IoT devices
+and does not specify how credentials are provisioned to devices.
+The WoT Architecture document does use a strict separation
+of private security data from public data and metadata, and
+recommends the use of an isolated private security data subsystem.
+
+The WoT Runtime and WoT Scripting API are defined in such a way
+that they do not have direct access to private credentials.
+Instead, an "abstract data type" is used in the WoT Protocol Bindings 
+to implement security operations such as authentication
+without revealing such private security data to WoT Applications.
+
+### Does this specification introduce new state for an origin that persists across browsing sessions? 
+
+This is not a browser-oriented specification although
+WoT Things can be accessed through browsers as if they
+were web services. 
+However, no new state is introduced or required in browsers beyond
+current mechanisms (cookies, etc).
+
+WoT Things can also act as clients, however, accessing
+web services and other WoT Things acting as servers.
+It is also possible that the same device could instantiate
+multiple WoT Runtimes.  However, there is noi explicit mechanism
```

Comment:
typo: noi

Reply:
What, noi is noi a word?   Anyhow... I will fix.   I guess running a spell checker over this document would not hurt...